src/index.js

```diff
@@ -45,6 +49,13 @@ import registerServiceWorker from "./registerServiceWorker";
 //  return INTEL;
 //};
 
+// Initialize time ago
+JavascriptTimeAgo.addLocale(en);
+
+const getIntel = () => {
+  return true;
+};
+
```

Comment:
Intel will be not required in [ops-front](https://github.com/snek-at/ops-front).

Reply:
```suggestion
```

Reply:
This is only a placeholder so this wonderful app does not crash on mi boi.